### ISSUE TITLE
Loading folders not completing for company mail on Office 365

### ISSUE THREAD

**Issue by WIP42** [3axap4eHko/OWA-pro#3]
Good day

This extension used to work with the same account, even after we moved our email to office365 recently; I had some issues and removed the account and tried to re-add it last week.

Ffter resolving issues getting the server settings to pass the config screen, specifically server address [https://outlook.com/$DOMAIN_NAME seemed correct] for anyone interested in office 365 settings, OWA-pro got to the [Select Folders] tab where it just keeps saying "Folders Loading...". Has anyone else experienced this and resolved it?

Thank you

**Comment by seqqueman** [comment]
I have the same problem, it never ends "loading"...

**Comment by Antecer** [comment]
the same problem also!

> I fill in this form with the following format：
chrome-extension://hldldpjjjagjfikfknadmnnmpbbhgihg/options.html#/accounts
name@outlook.com
https://outlook.live.com/mail
name@outlook.com
password

Then it will always Loading folders！

**Comment by akiirish** [comment]
Same.

**Comment by kirashi3** [comment]
Also experiencing "Loading Folders..." after signing in to an exchange account hosted by Microsoft.
Server Address: https://outlook.office365.com/owa/

**Comment by nykroy** [comment]
I have the same problem with a my profesional domain @xxxx.com

**Comment by FireWings2259** [comment]
Ok so I am having the same issue as well but after looking at it I am running Chrome Version 61.0.3163.100 (Official Build) (64-bit) and thus I believe this is the same issue as opened [Here](https://github.com/3axap4eHko/OWA-pro/issues/2). After pokening around I found that when attempting to GET the autodiscover.scv page (https://$YOUR_MAILDOMAIN/autodiscover/autodiscover.svc) I would get the error of "_Failed to load resource: net::ERR_BLOCKED_BY_CLIENT_" (This might be my browser but when I tested it with no other extensions I got the same) at line 41 of options.js. More importantly though the Exchange.asmx page ($YOUR_MAILDOMAIN/EWS/Exchange.asmx) **NOTE:** _there was no http/s prefix added to the console log of the error._, gave the same error as the autodiscover.svc page, this was also line 41 of option.js. The final error was line 50 of options.js, Giving an "_Uncaught (in promise) TypeError: Cannot convert undefined or null to object_" error at Native Object. Specifically from the "getXMLElementChildren" function in options.js line 41 at COLUMN **118074**, by "\<anonymous\>".

@3axap4eHko I do hope I gave relevant information.
Keep up the good work and Thanks,
Fire

EDIT - Fix layout.
